sklearnex/linear_model/logistic_regression.py

```diff
@@ -101,6 +98,8 @@ def __init__(
                 l1_ratio=l1_ratio,
             )
 
+        _onedal_cpu_fit = LogisticRegression_daal4py.fit
```

Comment:
It might be better to isolate next LogisticRegression_daal4py.fit code in separate hidden method with reuse in sklearnex to avoid double check of parameters and double decoration:
```python
            which, what = logistic_module, "_logistic_regression_path"
            replacer = logistic_regression_path
            descriptor = getattr(which, what, None)
            setattr(which, what, replacer)
            try:
                clf = LogisticRegression_original.fit(self, X, y, sample_weight)
            finally:
                setattr(which, what, descriptor)
            return clf
```
Link:
https://github.com/intel/scikit-learn-intelex/blob/089357705982b7aee8fe1c010166ec1599676f14/daal4py/sklearn/linear_model/logistic_path.py#L1027

Reply:
I agree with @Alexsandruss . #1668 will isolate this so that the double decoration will not occur, you won't have to add it to decorated_methods then. When it gets pulled in, you'll have to use 
```python
   fit = wrap_output_data(daal4py_fit)
```